java/ql/src/semmle/code/java/controlflow/internal/GuardsLogic.qll

```diff
@@ -40,19 +40,15 @@ predicate implies_v1(Guard g1, boolean b1, Guard g2, boolean b2) {
   )
   or
   exists(ConditionalExpr cond, boolean branch, BooleanLiteral boollit, boolean boolval |
-    cond.getTrueExpr() = boollit and branch = true
-    or
-    cond.getFalseExpr() = boollit and branch = false
+    cond.getBranchExpr(branch) = boollit
   |
```

Comment:
Style: Replace this `|` with `and` now that the first part is no longer a disjunction.